tools/public_api_guard/material/slide-toggle.d.ts

```diff
@@ -57,6 +57,7 @@ export declare class MatSlideToggleChange {
 }
 
 export interface MatSlideToggleDefaultOptions {
+    color?: ThemePalette;
```

Comment:
Won't we also need to update the docs for this change?

https://material.angular.io/components/slide-toggle/api#MatSlideToggleDefaultOptions

Reply:
The docs will be generated automatically from the source code.